manim/utils/color/core.py

```diff
@@ -665,9 +789,19 @@ def parse(
         ManimColor
             Either a list of colors or a singular color depending on the input
         """
-        if isinstance(color, (list, tuple)):
-            return [cls(c, alpha) for c in color]  # type: ignore
-        return cls(color, alpha)  # type: ignore
+
+        def isSequence(colors) -> TypeGuard[Sequence[ParsableManimColor]]:
```

Comment:
Maybe `is_sequence` and `is_parsable` to keep with the PEP8 naming standards?

Reply:
Whops